### PR TITLE
fix:failed to start nodeTopoInformer when tcp6 is not find in node

### PR DESCRIPTION
**Pull request by lucming**
### Ⅰ. Describe what this PR does

fix: https://github.com/koordinator-sh/koordinator/issues/743

### V. Checklist

- [ ] I have written necessary docs and comments
- [ ] I have added necessary unit tests and integration tests
- [ ] All checks passed in `make test`

### CODE REVIEW

File: pkg/util/system/common_linux.go [hunk 1]
```diff
@@ -52,14 +52,16 @@ type portAndPid struct {
...
-	if err != nil {
-		return nil, err
+
+	socks, err = netstat.TCP6Socks(fn)
+	if err == nil && socks != nil {
+		return socks, nil
 	}
-	return append(socks, v6Socks...), nil
+
+	return nil, err
```
Comment (eahydra) [inline pkg/util/system/common_linux.go:64]: Nice!  But both TCPSocks and TCP6Socks may return errors. It is recommended to return the aggregate  errors.

Comment (lucming) [reply pkg/util/system/common_linux.go]: > Nice! But both TCPSocks and TCP6Socks may return errors. It is recommended to return the aggregate errors.

thanks for reviewing.
done.

**Review by eahydra — APPROVED** [review]
/lgtm

**Review by jasonliu747 — APPROVED** [review]
/lgtm

**Pull request merged**